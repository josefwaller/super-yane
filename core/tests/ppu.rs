use insta::assert_binary_snapshot;
use paste::paste;
use super_yane::Console;
mod common;

rom_test! {background_basic, "./roms/8x8BGMap4BPP32x328PAL.sfc", 50_000}
