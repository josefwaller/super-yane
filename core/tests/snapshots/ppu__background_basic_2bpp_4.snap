---
source: core/tests/ppu.rs
expression: "c.ppu().screen_buffer.map(| b |\nb.to_le_bytes()).into_iter().flatten().collect()"
extension: bin
snapshot_kind: binary
---
